.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,10 @@
+
+---
+name: "\U0001F4DA Documentation"
```

Comment:
the mark is not set up properly.

Reply:
Thanks fixed.
https://github.com/pytorch/audio/blob/7149a3df3ab66bad265749110a27af42d48c3a78/.github/ISSUE_TEMPLATE/bug-report.md
https://github.com/pytorch/audio/blob/7149a3df3ab66bad265749110a27af42d48c3a78/.github/ISSUE_TEMPLATE/documentation.md
https://github.com/pytorch/audio/blob/7149a3df3ab66bad265749110a27af42d48c3a78/.github/ISSUE_TEMPLATE/feature-request.md
https://github.com/pytorch/audio/blob/7149a3df3ab66bad265749110a27af42d48c3a78/.github/ISSUE_TEMPLATE/questions-help-support.md